Sources/StreamVideoSwiftUI/Utils/Camera.swift

```diff
@@ -288,35 +277,12 @@ extension Camera: AVCaptureVideoDataOutputSampleBufferDelegate {
         from connection: AVCaptureConnection
     ) {
         guard let pixelBuffer = sampleBuffer.imageBuffer else { return }
-        
-        if
-            connection.isVideoOrientationSupported,
-            let videoOrientation = videoOrientationFor(deviceOrientation),
-            connection.videoOrientation != videoOrientation {
-            connection.videoOrientation = videoOrientation
+
+        let currentOrientation = orientationAdapter.orientation.captureVideoOrientation
```

Comment:
probably this is affecting the rotated camera in the lobby?

Reply:
Actually this PR contains the fix for the lobby preview